lib/tasks/assets.rake

```diff
@@ -0,0 +1,9 @@
+if Rails.env.production?
```

Comment:
should this be `unless Rails.env.assets?`

We should only be running this rake task in assets environment, right?